trieste/models/gpflow/sampler.py

```diff
@@ -220,8 +220,6 @@ def __init__(
         :param sample_size: The desired number of samples.
         :param model: The model to sample from.
         :param dataset: The data from the observer. Must be populated.
-        :sample_min_value: If True then sample from the minimum value of the function,
-            else sample the function's minimiser.
         :param num_features: The number of features used to approximate the kernel. We use a default
```

Comment:
(I'm disappointed this didn't give a docs build warning. I'll jot down a note to investigate.)